src/structure.js

```diff
@@ -17,7 +17,7 @@ export const structure = {
     },
     {
       name: 'Type',
-      components: ['Heading', 'Markdown', 'Paragraph', 'Text'],
+      components: ['Heading', 'Markdown', 'Paragraph', 'Text', 'Tag'],
```

Comment:
wondering about the category here -- it feels to me like it aligns more with the "Visualizations"

Reply:
It really didn't seem like a Visualization like a chart or graph. It seems more like Text treatment/styling and sometimes a Control. I don't see an obvious answer here yet.

Reply:
I think wherever "Tag" goes, then NameValueList should probably follow. Right now we have NameValueList in Visualizations but maybe it should change as well?

Reply:
Might be interesting to run a quick poll via Slack?

Reply:
I think running a poll is a good idea get everyone's thought but I agree where `tags` lives `nameValueList` should also live in the same place